src/App.tsx

```diff
@@ -1,173 +1,47 @@
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { getPeople } from './api';
+import { Person } from './types';
+import { HeadOfTable } from './components/TableHead/TableHead';
+import { Navigation } from './components/Navigation/Navigation';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+
+export const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople);
+  }, []);
+
+  return (
+    <div data-cy="app">
+      <HeadOfTable />
+      <main className="section">
+        <div className="container">
+          <Routes>
+            <Route path="/" element={<Navigation />}>
+              <Route index element={<h1 className="title">Home Page</h1>} />
+              <Route
+                path="*"
+                element={<h1 className="title">Page not found</h1>}
+              />
+              <Route
+                path="people"
+                element={<PeoplePage people={people} setPeople={setPeople} />}
+              >
+                <Route
+                  path=":personSlug"
+                  element={<PeoplePage people={people} setPeople={setPeople} />}
+                />
```

Comment:
Get and set people it's better to do only on their page, as you actually need them only on a specific page